Formula/watchman.rb

```diff
@@ -1,10 +1,17 @@
 class Watchman < Formula
   desc "Watch files and take action when they change"
   homepage "https://github.com/facebook/watchman"
-  url "https://github.com/facebook/watchman/archive/v2022.07.04.00.tar.gz"
-  sha256 "33f7e8e3c005ba2e0b81f6552df256d69aab057fb4b26dcd122fb662914b4b21"
   license "MIT"
-  head "https://github.com/facebook/watchman.git", branch: "main"
+
+  stable do
+    url "https://github.com/facebook/watchman/archive/v2022.07.18.00.tar.gz"
+    sha256 "e602bcb49309adc88367d5e24aa9314819b468ec79c8b7986f413bd3ea0d67e6"
+
+    resource "edencommon" do
+      url "https://github.com/facebookexperimental/edencommon.git",
+          revision: "1da3b6542ee1b2634a6108ad84bbc25b7f5785da"
+    end
```

Comment:
No tags/releases. I just went with commit that was close to other `v2022.07.18.00` releases. This particular commit is for file `folly-rev.txt` that matches the `folly` release in this PR.

Reply:
We should ask upstream to cut tags. They're usually pretty willing to do so.